Fix AbstractTrees download command for local run

aminya opened the pull request:
Fix AbstractTrees download command for local run
Fix running FemtoCleaner.cleanrepo command
Added Julia 6.4 download link
Fixed non-working URLs
Brought local running in Readme upfront

aminya committed:
Fix AbstractTrees download command for local run

Fix AbstractTrees download command for local run
Fix running FemtoCleaner.cleanrepo command
Added Julia 6.4 download link
Fixed non-working URLs
Brought local running in Readme upfront

logankilpatrick commented on README.md:
```diff
@@ -7,9 +7,39 @@ perform code formatting. The logic behind recognizing and rewriting deprecated c
 
+## Running FemtoCleaner locally
+
+It is possible to run FemtoCleaner locally (to fix, for example, deprecations in a private repository).
```
Please change this to "It is recommended that you run FemtoCLeaner locally". This is because the machine it was running on is dead so the actual app no longer works.

logankilpatrick reviewed (changes requested):
Please make those changes. I will take a stab at making this work with 1.0 and such. @Keno can you review thios when you have a chance.

aminya committed:
Update README.md

https://github.com/JuliaComputing/FemtoCleaner.jl/pull/112#discussion_r324008000

aminya committed:
Automated commit made by MassInstallAction.jl

aminya committed:
Automated commit made by MassInstallAction.jl

aminya committed:
Merge pull request #3 from aminya/massinstallaction/set-up-TagBot

MassInstallAction: Install the TagBot workflow on this repository

aminya committed:
Merge pull request #2 from aminya/massinstallaction/set-up-CompatHelper

MassInstallAction: Install the CompatHelper workflow on this repository
